.github/workflows/release_mac.yml

```diff
@@ -103,9 +103,8 @@ jobs:
         pytest
 
     - name: Verify ONNX with ONNX Runtime PyPI package
-      if: matrix.python-version != '3.11'
       run: |
-        python -m pip install -q onnxruntime
+        python -m pip install -q onnxruntime==1.13.1
```

Comment:
Freeze ORT version for now to unblock release pipeline failure on Mac. See related issue: https://github.com/microsoft/onnxruntime/issues/14663.